daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md

```diff
@@ -0,0 +1,583 @@
+﻿---
+type: docs
+title: "Zeebe command binding spec"
+linkTitle: "Zeebe command"
+description: "Detailed documentation on the Zeebe command binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/zeebe-commands/"
+---
+
+## Component format
+
+To setup Zeebe command binding create a component of type `bindings.zeebe.command`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+
+See [this](https://docs.camunda.io/docs/product-manuals/zeebe/zeebe-overview/) for Zeebe documentation.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: bindings.zeebe.command
+  version: v1
+  metadata:
+  - name: gatewayAddr
+    value: <host>:<port>
+  - name: gatewayKeepAlive
+    value: 45s
+  - name: usePlainTextConnection
+    value: true
+  - name: caCertificatePath
+    value: /path/to/ca-cert
+```
+
+## Spec metadata fields
+
+| Field                   | Required | Binding support |  Details | Example |
+|-------------------------|:--------:|------------|-----|---------|
+| gatewayAddr             | Y | Output | Zeebe gateway address                                                                     | `localhost:26500` | 
+| gatewayKeepAlive        | N | Output | Sets how often keep alive messages should be sent to the gateway. Defaults to 45 seconds  | `45s` | 
+| usePlainTextConnection  | N | Output | Whether to use a plain text connection or not                                             | `true,false` | 
+| caCertificatePath       | N | Output | The path to the CA cert                                                                    | `/path/to/ca-cert` | 
```

Comment:
How will this work on Kubernetes?

Reply:
You can mount a secret into the container with Kubernetes: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets

Reply:
Perfect, thanks!